CONTRIBUTING.md

```diff
@@ -66,13 +66,44 @@ with `fix:`, `feat:` or similar according to
 ## <a name="releasing">Releasing (maintainers only)</a>
 
 This repo uses [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
-and Github Actions for continuous deployment (CD).
+and GitHub Actions for continuous deployment (CD).
 
-1. Go to the [Github Actions tab](https://github.com/muxinc/media-chrome/actions), 
+> If you're unfamiliar with conventional commits, it's a good idea to review the link above before continuing.
+
+Here's a quick summary of how we use conventional commits in this repository:
+
+- Commit messages prefixed with `fix:` will notify CD that the release is minimally a `patch` release.
+- Commit messages prefixed with `feat:` will notify CD that the release is minimally a `minor` release.
+- Major releases must always be designated manually during the Release process.
+- All other conventional commits have no impact on the versioning.
+
+## Review commit messages since last release
```

Comment:
suggestion(non-blocking): Since these new sections are all subsections of `## <a name="releasing">Releasing (maintainers only)</a>`, above, you may want to make these new ones `###` sections instead